Containers/borgbackup/backupscript.sh

```diff
@@ -425,19 +527,29 @@ fi
 
 # Do the backup test
 if [ "$BORG_MODE" = test ]; then
-    if ! [ -d "$BORG_BACKUP_DIRECTORY" ]; then
-        echo "No 'borg' directory in the given backup directory found!"
-        echo "Only the files/folders below have been found in the given directory."
-        ls -a "$MOUNT_DIR"
-        echo "Please adjust the directory so that the borg archive is positioned in a folder named 'borg' inside the given directory!"
-        exit 1
-    elif ! [ -f "$BORG_BACKUP_DIRECTORY/config" ]; then
-        echo "A 'borg' directory was found but could not find the borg archive."
-        echo "Only the files/folders below have been found in the borg directory."
-        ls -a "$BORG_BACKUP_DIRECTORY"
-        echo "The archive and most importantly the config file must be positioned directly in the 'borg' subfolder."
-        exit 1
-    elif ! borg list "$BORG_BACKUP_DIRECTORY"; then
+    if [ -n "$BORG_REMOTE_REPO" ]; then
+        if ! borg info; then
```

Comment:
```suggestion
        if ! borg info &>/dev/null; then
```

Reply:
I think at least in the case, when doing the backup check, seeing the output could be useful in case it fails. It might show info about whether it's corrupt, can't connect to it, ...

Reply:
How about I just repeat the `borg info` in case it fails; see last commit. Then it only spams when it may be useful. There's probably a nicer way of doing this with `x=$(borg info)` and check `$?` or something instead of calling it again but this way it should work at least.

Reply:
`borg info >/dev/null` should do what you try to do. It only prints out if it fails.

Reply:
(Removing thr & from the command does this)

Reply:
Ah indeed, borg splits stdout/stderr nicely, I'll do that instead